### PR TITLE
Remove duplicated JWT from response

### LINKED ISSUE #25: Duplicate JWT on response

### PR DESCRIPTION
**Pull request by weslenng**
Fix #25

### CODE REVIEW

File: test/controllers/auth.test.js [hunk 1]
```diff
@@ -65,31 +65,32 @@ describe(path, () => {
       expect(status).to.equal(200);
     });
 
-    it('should return a token', async () => {
-      const { body } = await request(server)
+    it('should have Set-Cookie header', async () => {
```
Comment (joviane) [inline test/controllers/auth.test.js]: I think this test should verify if there is a jwt in Set-Cookie header. Doesn't make sense to me to verify if only the header is present.

File: test/controllers/auth.test.js [hunk 2]
```diff
@@ -65,31 +65,32 @@ describe(path, () => {
...
 
-      expect(body).to.have.property('jwt');
-      expect(body.jwt).to.match(/^([^.]+\.){2}[^.]+$/);
+      expect(headers).to.have.property('set-cookie');
+      expect(headers['set-cookie']).to.be.a('array');
+      expect(headers['set-cookie'][0]).to.be.a('string');
     });
 
-    it('should have Set-Cookie with SameSite=Strict; Secure; HttpOnly and token', async () => {
+    it('should Set-Cookie have jwt; SameSite=Strict; Secure; HttpOnly attribs', async () => {
```
Comment (joviane) [inline test/controllers/auth.test.js]: This test validates if Set-Cookie has jwt, if it was in the correct format and the other attributes. In this case is best to separate the tests in two: one being the first comment (Set-Cookie having jwt and it is correct) and the other if Set-Cookie has the other attributes.

**Review by joviane — CHANGES_REQUESTED** [review]
Please check out the comments below and also create a feature branch instead of pushing your develop branch,

File: test/controllers/auth.test.js (hunk 1 above)
Comment (masterl) [reply test/controllers/auth.test.js]: The next test does that.
He just separated testing the existence of the header from testing the actual content.
Both are needed, in my opinion.

File: test/controllers/auth.test.js (hunk 2 above)
Comment (weslenng) [reply test/controllers/auth.test.js]: Done

**Review by joviane — APPROVED** [review]

**Pull request closed without merging**